keps/prod-readiness/sig-node/2872.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2872
+alpha:
+  approver: 
```

Comment:
I have to review this one anyways, please set me as PRR approver

Reply:
Thanks @ehashman